### PR TITLE
build: update grammY peer dep to ^1.20.1

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Reported in https://t.me/grammyjs/193250

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,20 +25,17 @@
...
+        "deno2node": "^1.11.0"
     },
     "files": [
         "out/"
     ],
     "main": "out/mod.js",
     "type": "commonjs",
-    "exports": {
-        ".": "./out/mod.js"
-    },
```
Comment (KnorpelSenf) [inline package.json:40]: This needs to be removed in order to to use `grammy/types` on Node. Why???

Comment (roziscoding) [reply package.json:40]: doesn't it break ESM or something if we remove this?

**Review by roziscoding — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (KnorpelSenf) [reply package.json:40]: Good point, we should try. Conversations with without it, though.

**Comment by KnorpelSenf** [comment]
I guess we should wait for @wojpawlik

File: package.json (hunk 1 above)
Comment (wojpawlik) [reply package.json:40]: I find lack of `"types"` suspicious... not sure if that's the issue tho.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
         "url": "https://github.com/grammyjs/hydrate/issues"
     },
     "scripts": {
+        "prepare": "npm run build",
         "build": "deno2node tsconfig.json",
```
Comment (wojpawlik) [inline package.json:20]:
```suggestion
        "build": "deno2node",
```
for forward compatibility with Deno2node 2.0.

**Comment by KnorpelSenf** [comment]
This does not fix anything, but at least it updated the versions of all deps so I would like to merge anyway.

**Pull request merged**